storage_service/common/management/commands/create_user.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand, CommandError
+
+from tastypie.models import ApiKey
+
+
+class Command(BaseCommand):
+    help = 'Create user with all the arguments specified. ' \
+        'If the user already exists its attributes will be updated.'
+    args = ('username', 'email', 'password', 'api_key')
+
+    def add_arguments(self, parser):
+        parser.add_argument('--username', required=True)
+        parser.add_argument('--email', required=True)
+        parser.add_argument('--password', required=True)
+        parser.add_argument('--api-key', required=True)
```

Comment:
Awesome! That will need consideration when merging back into jisc, since the command will no longer create superuser by default. Which I think is the right thing to do.